server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/cs/factory/DeployedCacheStoreFactory.java

```diff
@@ -18,14 +17,22 @@
  */
 public class DeployedCacheStoreFactory implements CacheStoreFactory {
 
-   private Map<String, DeployedCacheStoreMetadata> deployedCacheStores = Collections.synchronizedMap(new HashMap<String, DeployedCacheStoreMetadata>());
+   private static final int TIMEOUT = 20;
+   private static final TimeUnit TIMEOUT_UNITS = TimeUnit.SECONDS;
```

Comment:
I think it would be better to rename `TIMEOUT` to `TIMEOUT_SECONDS` and use `TimeUnit.SECONDS` directly in the method.